Scientific_workflows/FractionalCoverOfWaterProject/Manual_classification_workflow.ipynb

```diff
@@ -0,0 +1,994 @@
+{
```

Comment:
<p>More comments for what each of these cells do. Just something simple - load modules, set up Dask, set up Bokeh, etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/663/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done.</p>